bindings/c/test/mako/mako.cpp

```diff
@@ -2217,7 +2244,7 @@ void printReport(Arguments const& args,
 	putTitle("Errors");
 	first_op = true;
 	for (auto op = 0; op < MAX_OP; op++) {
-		if (args.txnspec.ops[op][OP_COUNT] > 0 && op != OP_TRANSACTION) {
+		if ((args.txnspec.ops[op][OP_COUNT] > 0 && op != OP_TRANSACTION) || op == OP_COMMIT) {
```

Comment:
`args.txnspec.ops[OP_COMMIT][OP_COUNT]` might be 0. I added it because I wanted to print the number of timeouts during commits in the next line (that's where most timeouts occur). 

```
                      GET       UPDATE       INSERT       COMMIT          TPS  Conflicts/s 
============ ============ ============ ============ ============ ============ ============ 
Total OPS          375183        46892        46892         9607        96.04         2.19 
Errors                  0            0            0            0 
Timeouts               25            0            0        37066 


```

Reply:
OK, I understand now. It is checking the transaction spec, and commits are never a part it